loops-arrays-objects.js

```diff
@@ -32,8 +35,21 @@ const scores = [
 // D
 // ==========================================
 
-
-
+// console.log("Opdracht 1b");
+//
+// for (let i = 0; i < scores.length; i++) {
+//     if (scores[i].score < 60) {
+//         console.log("F");
+//     } else if (scores[i].score >= 60 && scores[i].score < 70) {
```

Comment:
overbodige code want <70 is automatisch >= 60 omdat wanneer de waarde op 60 of lager komt, er al wat anders geprint wordt.